plugins/chat/spec/fabricators/chat_fabricator.rb

```diff
@@ -86,17 +86,26 @@
     Group.refresh_automatic_groups!
     channel.add(user)
 
-    resolved_class.call(
-      chat_channel_id: channel.id,
-      guardian: user.guardian,
-      message:
-        transients[:message] ||
-          Faker::Lorem.paragraph_by_chars(number: 500).gsub("...", "…").gsub("..", "…"),
-      thread_id: transients[:thread]&.id,
-      in_reply_to_id: transients[:in_reply_to]&.id,
-      upload_ids: transients[:upload_ids],
-      incoming_chat_webhook: transients[:incoming_chat_webhook],
-    ).message
+    result =
+      resolved_class.call(
+        chat_channel_id: channel.id,
+        guardian: user.guardian,
+        message: transients[:message] || Faker::Lorem.words(number: 5).join(" "),
+        thread_id: transients[:thread]&.id,
+        in_reply_to_id: transients[:in_reply_to]&.id,
+        upload_ids: transients[:upload_ids],
+        incoming_chat_webhook: transients[:incoming_chat_webhook],
+        process_inline: true,
+      )
+
+    if result.failure?
+      raise RSpec::Expectations::ExpectationNotMetError.new(
+              "Service `#{resolved_class}` failed, see below for step details:\n\n" +
+                result.inspect_steps.inspect,
+            )
+    end
```

Comment:
It feels cleaner to show the steps when a failure is due to an unexpected service failure, it will make writing specs easier

Reply:
This is how a failure looks now btw:

![Screenshot 2023-11-05 at 21 09 18](https://github.com/discourse/discourse/assets/339945/0a8b0295-29e6-469d-8d13-6b61a25cb3ac)